attendance/lib/homePage/home_page_viewmodel.dart

```diff
@@ -11,7 +11,7 @@ import '../scan_exceptions.dart';
 
 
 abstract class HomePageViewModel extends State<HomePage> {
-  String scanResult = "";
+  String scanResult = 'Scan Error: Make sure you\'re scanning the right code';
```

Comment:
@Bahaaib Use double quotes instead of single quotes if the string contains a single quote and avoid `\'`

Reply:
Is that in-house rules or came out for a reason?